src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  margin-top: 20px;
+
+  width: $calendar-width;
+
+  &__field {
+    position: relative;
+
+    justify-content: center;
+    align-items: center;
+
+    width: $block-size;
+    height: $block-size;
+    margin: 0 $block-gap $block-gap 0;
```

Comment:
As I understand in this task we can use "gap". It's more simple, just "gap: $block-gap"
![image](https://user-images.githubusercontent.com/91382085/138101449-d317ece4-4ec8-4780-8372-9ef6b63e020f.png)